lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -78,33 +80,36 @@ class ReportUIFeatures {
    * @param {LH.Result} report
    */
   initFeatures(report) {
-    if (this._dom.isDevTools()) return;
-
     this.json = report;
-    this._setupMediaQueryListeners();
-    this._setupExportButton();
+
+    if (this._dom.isDevTools()) {
+      // Not ready for DevTools.
+      // Saving as HTML save the entire AuditsPanel.
```

Comment:
hm it's not *super* clear what's being enabled in DT and what's not since the comments sorta line up with what's in the `else`.

not sure how to fix this maybe it's just moving to have a comment for each line explaining why it can't be enabled and/or when it might be moved out of the `else`?

Reply:
sure - `...does not work.` is not a good comment :)